Frameworks/CoreGraphics/CGContext.mm

```diff
@@ -2356,14 +2360,105 @@ void CGContextDrawTiledImage(CGContextRef context, CGRect rect, CGImageRef image
 
 #pragma region Drawing Operations - Gradient + Shading
 /**
- @Status Stub
+* Insert a transparent color at the specified 'location'.
+* This will also move the color at the specified 'location' to the supplied 'position'
+*/
+static inline void __CGGradientInsertTransparentColor(std::vector<D2D1_GRADIENT_STOP>& gradientStops, int location, float position) {
```

Comment:
This does not appear to insert them in order. Are we required to do that for D2D's sake?